container-registry/container-analysis/src/test/java/com/example/containeranalysis/SamplesTest.java

```diff
@@ -0,0 +1,256 @@
+/*
+ * Copyright 2018 Google Inc.
```

Comment:
Should be `Google LLC` for all new headers

Reply:
The checkstyle plugin seems to enforce using Inc for me. Am I using an outdated version or something?

Reply:
Make sure the following parent is in your pom:
```
    <!--
      The parent pom defines common style checks and testing strategies for our samples.
      Removing or replacing it should not affect the execution of the samples in anyway.
    -->
    <parent>
        <groupId>com.google.cloud.samples</groupId>
        <artifactId>shared-configuration</artifactId>
        <version>1.0.9</version>
    </parent>
```

Its should accept `Inc.` or `LLC` - no punctuation on the second.

Reply:
I do have that in my pom, but it looks like LLC works if I use 1.0.10 instead, so I updated it

Reply:
Ah that's weird. We must have some other out of dates in this repo.